week3/dainn/todo/app.js

```diff
@@ -0,0 +1,112 @@
+const todoListEl = document.getElementById("todoList");
+const todoInputEl = document.getElementById("todoInput");
+
+const API_URL = "http://localhost:3000/todos";
+
+fetch(API_URL)
+  .then((response) => response.json())
+  .then((data) => renderTodo(data));
+
+const updateTodo = (todoId, originalTitle) => {
+  console.log('updateTodo');
+  const todoItem = document.querySelector(`#todo-${todoId}`);
+  // mission
+  console.log(todoItem);
+  console.log(todoItem.id);
+
+  const listEl = document.getElementById(todoItem.id);
+  const input = document.createElement('input');
+  input.type = 'text';
+  input.placeholder = todoItem.title;
```

Comment:
placeholder는 string인데, todoItem은 DOM객체이기 때문에 title이라는 속성이 없습니다!
```suggestion
  input.placeholder = originalTitle;
```
이렇게하면 원래 할 일이 placeholder로 잘 보입니다.